motionlayout/src/main/java/com/google/androidstudio/motionlayoutexample/Consts.kt

```diff
@@ -0,0 +1,8 @@
+package com.google.androidstudio.motionlayoutexample
+
+object Consts {
+
+    // Tentative variable to make it explicit that MotionLayout#setDebugMode(2 (or above)) works
+    // as the same as MotionLayout#setShowPaths(true) in the alpha2 or below
+    const val SHOW_PATHS = 2
```

Comment:
Currently, Debug inside MotionLayout is compared with int. However, how about using fields like `DEBUG_SHOW_NONE / DEBUG_SHOW_PROGRESS / DEBUG_SHOW_PATH`?

Reply:
I think those constants should be in the library instead of in the sample.
So we don't have to introduce variables not used in the samples.

@camaelon @jafu888 how do you think?

Reply:
Sorry. I already wanted to tell you that that field is in the library.
<img width="528" alt="2019-01-05 12 11 00" src="https://user-images.githubusercontent.com/1534926/50719668-2a037100-10e3-11e9-865b-2efadf65bc40.png">

Reply:
@Pluu oh sorry, you are totally right. Deleted the Consts class in the sample.